component/prometheus/remotewrite/types.go

```diff
@@ -204,6 +204,14 @@ func convertConfigs(cfg Arguments) (*config.Config, error) {
 			Name:                rw.Name,
 			SendExemplars:       rw.SendExemplars,
 
+			// SendNativeHistograms is always set to true. Since scraping native
+			// histograms happens transparently from supported targets, native
+			// histograms should always be best-effort remote_written. Otherwise, if
+			// native histograms were controllable, then users might be surprised
+			// when some metrics from targets were never sent to the remote_write
+			// endpoint.
+			SendNativeHistograms: true,
```

Comment:
I haven't taken the time to search for the answer yet, but I'm wondering. Until histograms are supported in the remote_write, will this just log a few more lined, or is it possible that it can lead to decoding errors that will impact other metrics being sent?

Reply:
Yeah, I'm not sure, I guess I'll make it configurable and disable it by default for now since it's bleeding edge.